lib/vicadmin/validate.go

```diff
@@ -130,41 +130,41 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 		v.FirewallIssues = template.HTML("")
 		v.LicenseStatus = BadStatus
 		v.LicenseIssues = template.HTML("")
-	} else {
+		return nil
```

Comment:
Returning `nil` here causes a panic on https://github.com/vmware/vic/blob/master/cmd/vicadmin/server.go#L560 when the session is nil.